technical-reports/color/token-naming.md

```diff
@@ -0,0 +1,393 @@
+# Token naming
+
+## Categorization
+
+There are 3 main categories of design tokens that we will continue to reference in this specification.
+
+### Base
+
+Base tokens are the lowest level tokens and typically consist of a name and [hexadecimal](https://www.w3.org/TR/css-color-4/#hex-notation) value pair.
```

Comment:
This references a hex value pair, but the example below doesn't match that description.

Reply:
good catch! I'll fix.